### PR TITLE
Upgrade eslint, remove nsp

### PR DESCRIPTION
**Pull request by kendaleiv**
Fixes reported vulnerability.

### CODE REVIEW

File: build.cmd [hunk 1]
```diff
@@ -12,9 +12,6 @@ if %ERRORLEVEL% neq 0 goto BuildFail
 call npm run validate
 if %ERRORLEVEL% neq 0 goto BuildFail
 
-call npm run audit
-if %ERRORLEVEL% neq 0 goto BuildFail
-
```
Comment (tgharold) [inline build.cmd]: Why this change?

Comment (kendaleiv) [reply build.cmd]: https://www.npmjs.com/package/nsp has been deprecated.
